admin/src/app/(auth)/osa/page.tsx

```
@@ -0,0 +1,1208 @@
+          <div className="grid grid-cols-2 md:grid-cols-4 gap-3">
+            {Object.entries(categoryInfo).map(([key, info]) => {
+              const count = stats.byCategory[key] || 0;
+              if (count === 0) return null;
+              const entries = activityLog.filter((a) => a.category === key);
+              return (
+                <div
+                  key={key}
+                  className="relative group p-3 rounded-lg border-2 cursor-pointer transition-all hover:shadow-md"
+                  style={{ borderColor: info.color + "44", backgroundColor: info.color + "08" }}
+                  onClick={() => setSelectedEntry(entries[0] || null)}
+                >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)